lib/gruf/controllers/base.rb

```diff
@@ -73,6 +73,7 @@ def call(method_key, &block)
       rescue GRPC::BadStatus
         raise # passthrough
       rescue StandardError => e
+        Gruf.logger.error "Unhandled #{e.class.name}: #{e.message} #{e.backtrace.join("\n")}"
```

Comment:
So I think this should go into the RequestLogging interceptor, and not here. You likely just need to rescue and re-raise StandardError there.

Reply:
Also - you may want to truncate the backtrace to the last 5-8 lines, so it's not huge.

Reply:
This is a global exception handler for all code including any other interceptors that may be added, so I think it's useful to have it here...

Reply:
Right, but having it here makes it not easily configurable (as opposed to the interceptor) or overridable in how logs are made. I'd rather push as much logging to the interceptors themselves to allow them to be fully configurable.

Reply:
If I could guarantee that this error handler were the outermost interceptor, we could put all of this rescue logic there. What do you think about that? Create a new interceptor that is at the outermost layer, and move the begin...rescue logic there...

Reply:
@mattolson So you can do that today! Interceptor runtime is controlled entirely from configuration: https://github.com/bigcommerce/gruf#configuring-interceptors

They execute in a FIFO manner, so you can manage this as you want.